packages/core/admin/ee/server/bootstrap.js

```diff
@@ -20,7 +20,10 @@ module.exports = async () => {
     await actionProvider.registerMany(actions.auditLogs);
   }
 
-  if (features.isEnabled('review-workflows')) {
+  if (
+    features.isEnabled('review-workflows') &&
+    strapi.config.get('admin.reviewWorkflows.enabled', true)
```

Comment:
Shouldn't `features.isEnabled('review-workflows')` return false if we have `admin.reviewWorkflows.enabled === false` ? 🤔

Reply:
not really :/ , the features.isEnabled uses the license registry, so it only checks if the feature is enabled in the registry , not any other config.

Reply:
omw! 🏃